servant-swagger/servant-swagger.cabal

```diff
@@ -20,7 +20,7 @@ description:
   * and [many others](http://swagger.io/open-source-integrations/).
 homepage:            https://github.com/haskell-servant/servant/servant-swagger
 bug-reports:         https://github.com/haskell-servant/servant/issues
-license:             BSD3
+license:             BSD-3-Clause
```

Comment:
For the record : Why BSD3 license has to be changed in this PR?

Reply:
Because using such a Cabal file format mandates the usage of SPDX-compliant identifiers, which are listed at https://spdx.org/licenses.

Reply:
i think both are supported so it could remain BSD3 but I believe the intent of cabal is to favor the SPDX format so I think it's fine to change it here. I haven't changed it everywhere though as I prefer to keep changes minimal.

Reply:
I'be more comfortable if you got that out of the way in this PR (maybe in a different commit, for ease of review). The next versions of Cabal will force us to comply

Reply:
done